config/env_sample.json

```diff
@@ -93,19 +93,19 @@
     "/*  LTI is disabled by default": "*/",
     "STUDENT_DASHBOARD_LTI": false,
     "/*  LTI 1.3 configuration":"*/",
-    "/*  The first key of LTI_CONFIG is your Canvas domain name":"*/",
+    "/*  The first key of LTI_CONFIG is your environment specific Canvas url and not domain specific":"*/",
     "LTI_CONFIG": {
         "https://canvas.instructure.com": [
             {
                 "/*The default set of variables for LTI validation":"*/",
                 "default": true,
                 "/*LTI Dev Key from Canvas":"*/",
                 "client_id": "17700000000000111",
```

Comment:
The comments below are now too specific and confusing.  Not everyone who may set up MyLA knows the difference between domain-specific and environment-specific.

We should simply give examples of the hosts allowed in the URLs.